shell/src/main/java/alluxio/shell/command/MountCommand.java

```diff
@@ -97,15 +115,15 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
       Properties properties = cl.getOptionProperties(OPTION_OPTION.getLongOpt());
       options.setProperties(Maps.fromProperties(properties));
     }
-
     mFileSystem.mount(alluxioPath, ufsPath, options);
     System.out.println("Mounted " + ufsPath + " at " + alluxioPath);
     return 0;
   }
 
   @Override
   public String getUsage() {
-    return "mount [--readonly] [--shared] [--option <key=val>] <alluxioPath> <ufsURI>";
+    return "mount [--readonly] [--shared] [--option <key=val>] <alluxioPath> <ufsURI>\n"
+            + "mount [--readonly] [--shared] [--option <key=val>]";
```

Comment:
just "mount". the part of "[--readonly] [--shared] [--option <key=val>]" does not make sense when mounting without args